src/shared/hooks/use-linked-interactives-authoring.tsx

```diff
@@ -0,0 +1,106 @@
+import { useEffect, useState } from "react";
+import deepmerge from "deepmerge";
+import { JSONSchema6 } from "json-schema";
+import {
+  useAuthoredState, useInitMessage, getInteractiveList, ILinkedInteractive, setLinkedInteractives
+} from "@concord-consortium/lara-interactive-api";
+
+export interface ILinkedInteractiveProp {
+  label: string;
+  supportsSnapshots?: boolean;
+}
+
+export interface IProps {
+  linkedInteractiveProps?: ILinkedInteractiveProp[];
+  schema: JSONSchema6;
+}
+
+type AuthoredState = {[key: string]: any};
```

Comment:
I believe `Record<string, unknown>` or `Record<string, any>` is the recommended way to type an arbitrary object at this point.

Reply:
Thanks, I've updated it.